server/lib/schema/index.ts

```diff
@@ -2113,6 +2125,98 @@ const schema = new GraphQLSchema({
           },
         ),
       },
+      updateTreeExperts: {
+        type: GraphQLBoolean,
+        args: {
+          bulkTreeInputs: { type: new GraphQLList(TreeReplaceExpertInput) },
+        },
+        resolve: requireAdmin(
+          "You must be logged in as an admin to update tree experts",
+          async (_, { bulkTreeInputs }, context) => {
+            let transaction;
+            try {
+              transaction = await sequelize.default.transaction();
+              for (const treeInput of bulkTreeInputs) {
+                const {
+                  treeId,
+                  emailsOfHonestOracles,
+                  emailsOfMaliciousOracles,
+                  replaceOracles,
+                } = treeInput;
+
+                const tree: Tree = await Tree.findByPk(treeId);
+                if (tree === null) {
+                  throw new Error(`No tree found for TreeId ${treeId}`);
+                }
+
+                if (replaceOracles) {
+                  await UserTreeOracleRelation.destroy({
+                    where: {
+                      TreeId: tree.id,
+                    },
+                    transaction,
+                  });
+                }
+
+                for (const email of emailsOfHonestOracles) {
+                  const user: User | null = await User.findOne({
+                    where: {
+                      email: email.toLowerCase(),
```

Comment:
I'll make an issue about using something other than `String.prototype.toLowerCase` here. Maybe `Sequelize.Op.iLike` which is based on Postgres `ILIKE`? Again, no need to do anything this PR.